primers.go

```diff
@@ -103,3 +104,79 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
+Start of the De Bruijn stuff
+******************************************************************************/
+
+// DeBruijn generates a DNA DeBruijn sequence
+// https://rosettacode.org/wiki/De_Bruijn_sequences#Go
```

Comment:
Can you add this reference link and write a thing in line on De Bruijn sequences an how they work? https://en.wikipedia.org/wiki/De_Bruijn_sequence

Reply:
Gotcha. Added.